### PR TITLE
Feature/fix lazy saving

### PR DESCRIPTION
**Pull request by michal1990**

### CODE REVIEW

File: app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/TagsManager.java [hunk 1]
```diff
@@ -41,17 +41,7 @@ public TagsManager(ExifInterface exifInterface) {
     boolean editTag(Tag tag) {
```
Comment (kejmil01) [inline app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/TagsManager.java]: Why is this method boolean, when it is always returning true value?

File: app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/TagsManager.java [hunk 2]
```diff
@@ -105,9 +95,8 @@ boolean clearTag(Tag tag) {
     public boolean clearTags(List<Tag> tagsList) {
```
Comment (kejmil01) [inline app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/TagsManager.java]: Why is this method boolean, when it is always returning true value?

Comment (BartSoftware) [reply app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/TagsManager.java]: +1

File: app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/TagsManager.java (hunk 1 above)
Comment (BartSoftware) [reply app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/TagsManager.java]: +1

**Review by BartSoftware — CHANGES_REQUESTED** [review]

**Review by kejmil01 — APPROVED** [review]

**Review by BartSoftware — APPROVED** [review]

**Pull request merged**